test/suites/crud_server.lua

```diff
@@ -18,51 +69,19 @@ box.schema.user.grant(
     'read,write,execute',
     'universe'
 )
-box.schema.create_space(
- 'tester', {
-  format = {
-    {name = 'id', type = 'unsigned'},
-    {name = 'bucket_id', type = 'unsigned'},
-    {name = 'name', type = 'string'},
-  }
-})
-box.space.tester:create_index('primary_index', {
-    parts = {
-        {field = 1, type = 'unsigned'},
-    },
-})
-box.space.tester:create_index('bucket_id', {
-    parts = {
-        {field = 2, type = 'unsigned'},
-    },
-    unique = false,
-})
-
--- Setup vshard.
-_G.vshard = vshard
-box.once('guest', function()
-    box.schema.user.grant('guest', 'super')
-end)
-local uri = 'guest@0.0.0.0:' .. primary_listen
-local cfg = {
-    bucket_count = 300,
-    sharding = {
-        [box.info().cluster.uuid] = {
-            replicas = {
-                [box.info().uuid] = {
-                    uri = uri,
-                    name = 'storage',
-                    master = true,
-                },
-            },
-        },
-    },
-}
-vshard.storage.cfg(cfg, box.info().uuid)
-vshard.router.cfg(cfg)
-vshard.router.bootstrap()
 
--- Initialize crud.
-crud.init_storage()
-crud.init_router()
-crud.cfg{stats = true}
+if crud_imported == false or vshard_imported == false then
+    local fail_msg = 'The crud/vshard modules are not detected, ' ..
+                     'installation via rocks install is required ' ..
+                     'for CRUD testing purposes. You can use ' ..
+                     '<tarantoolctl rocks install crud> or ' ..
+                     '<tt rocks install crud> to install modules'
+    -- The print output will be captured in the logs.
+    print(fail_msg)
+    -- Because of the implementation of the testing framework,
+    -- this will cause the crash of the CRUD test and display to user
+    -- a message about require fail during instance configuration.
+    box.info.status = fail_msg
```

Comment:
This hack is definitely not nice. Let's do something other here, if possible

Reply:
Unfortunately, I don't see any other options yet. The [design](https://github.com/tarantool/tarantool-python/blob/master/test/suites/lib/tarantool_server.py#L237) of the test framework, as far as I can see, does not allow you to get a more accurate result without test framework changes. It may make sense to implement exit conditions from waiting in `wait_until_started`. I'll try to consider this

Reply:
Well, if you don't find anything better in reasonable time (not more than a couple of hours of work), let's leave it as is for now.

Reply:
But I'm not sure this will work fine since `box.info.status` is highly likely to get overwritten by Tarantool

Reply:
> Let's do something relatively easy here, like pcalling `crud` in init file and checking in `setUp` if require is ok and failing (or skipping) otherwise

Done, if the solution is suitable, I will make the correct division into commits

Reply:
Yeah, I think it's suitable, thank you.